File: my_robot_tutorials/scripts/robot_news_radio_transmitter.py
#!/usr/bin/env python3

import rospy
from std_msgs.msg import String
if __name__ == '__main__':

    rospy.init_node('robot_news_radio_transmitter_py', anonymous=True)
    
    pub = rospy.Publisher("/robot_news_radio_py", String, queue_size=10)
    
    rate = rospy.Rate(2)
    
    while not rospy.is_shutdown():
        msg = String()    
        msg.data = "Hi, this kong from the robot news radio!"
        pub.publish(msg)
        rate.sleep()
        
    rospy.loginfo("Node was stopped")

File: my_robot_tutorials/src/robot_news_radio_transmitter.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main (int argc, char **argv)
{
    ros::init(argc, argv, "robot_news_radio_transmitter_cpp");
    ros::NodeHandle nh;

    ros::Publisher pub = nh.advertise<std_msgs::String>("/robot_news_radio_cpp", 10);

    ros::Rate rate (3);

    while (ros::ok()){
        std_msgs::String msg;
        msg.data = "Hi, This is Kong from the robot news Radio";
        pub.publish(msg);
        rate.sleep();


    }


}